payments/management/commands/calculate.py

```diff
@@ -10,9 +11,10 @@
 class Command(BaseCommand):
 
     def add_arguments(self, parser):
-        parser.add_argument('amount', nargs='+', type=int)
+        parser.add_argument('amount', nargs='+', type=float)
 
     def handle(self, *args, **options):
             amount = options['amount'][0]
+            amount = Decimal(amount)
```

Comment:
Could this be combined with the previous line?
```
amount = Decimal(options['amount'][0])
```
?

And/or, we probably need to check that amount had only a single argument before we look at the `0`th element of it?